dev-python/uvloop/uvloop-0.5.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_5 )
+inherit distutils-r1
+
+DESCRIPTION="Ultra-fast implementation of asyncio event loop on top of libuv."
+HOMEPAGE="https://github.com/magicstack/uvloop"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="doc examples test"
+
+CDEPEND=">=dev-libs/libuv-1.9.0:="
+DEPEND="
+	${CDEPEND}
```

Comment:
ditch `CDEPEND`. Put the `CDEPEND` in `RDEPEND`, and include `RDEPEND` in `DEPEND`